### PR TITLE
Updated examples

### PR DESCRIPTION
**Pull request by forsyth2**
Updated examples for Cori -- https://e3sm-project.github.io/e3sm_diags/_build/html/master/examples.html#running-the-examples

### CODE REVIEW

File: docs/source/examples.rst [hunk 1]
```diff
@@ -147,6 +147,49 @@ default variables. For each of the plot sets, the user needs to make a ``*_obs_v
 
 Viewing the Results
 -------------------
-Results from running all the examples can be found `here <https://portal.nersc.gov/project/e3sm/forsyth/examples/>`__.
-You can navigate to ``https://portal.nersc.gov/project/e3sm/forsyth/examples/<example directory>/viewer/`` to
+Results from running all the examples can be found `here <https://portal.nersc.gov/cfs/e3sm/forsyth/examples/>`__.
+You can navigate to ``https://portal.nersc.gov/cfs/e3sm/forsyth/examples/<example directory>/viewer/`` to
```
Comment (forsyth2) [inline docs/source/examples.rst:151]: Note: wait to merge until checking the correct path; see discussion on https://github.com/E3SM-Project/mache/issues/52

**Review by chengzhuzhang — APPROVED** [review]
Hi Ryan, Thank you for recreating the examples and this PR.

**Comment by forsyth2** [comment]
Note: #619 needs to be resolved before this can be merged.

**Comment by forsyth2** [comment]
Note: both #619 and #621 had to be merged for the checks on this pull request to work.

**Pull request merged**